api.planx.uk/modules/file/file.test.ts

```diff
@@ -41,39 +42,55 @@ describe("File upload", () => {
   });
 
   describe("Private", () => {
-    const ENDPOINT = "/private-file-upload";
+    const ENDPOINT = "/file/private/upload";
+    const auth = authHeader({ role: "teamEditor" });
+
+    it("returns an error if authorization headers are not set", async () => {
+      await supertest(app)
+        .post("/flows/1/move/new-team")
+        .expect(401)
+        .then((res) => {
+          expect(res.body).toEqual({
+            error: "No authorization token was found",
+          });
+        });
+    });
 
     it("should not upload without filename", async () => {
       await supertest(app)
         .post(ENDPOINT)
+        .set(auth)
         .field("filename", "")
         .attach("file", Buffer.from("some data"), "some_file.txt")
-        .expect(422)
+        .expect(400)
```

Comment:
One issue with the current simplified Zod error handling is that the validation checker in `api.planx.uk/shared/middleware/validate.ts` just returns a 400 so we lose some nuance. 

I'll open a GH issue describing this small improvement as well as nicer error formatting using something like https://github.com/causaly/zod-validation-error maybe.